src/pages/news/postgraphile-version-4-5.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+path: /news/postgraphile-version-4-5/
+title: PostGraphile Releases Version 4.5
+noToc: true
+---
+
+_Announced 2019-11-24_
+
+<p class='intro'>
+Graphile is pleased to announce the release of PostGraphile version 4.5.0, an upgrade recommended for all PostGraphile users.
+</p>
+<strong>🚨Beware:</strong> previously the CLI would ignore arguments it didn't understand, now it exits stating the arguments it wasn't expecting. If you were relying on this old behaviour, you will need to update your code to pass valid arguments.
+
+### New Feature: Explain in Graph*i*QL
+
+The headline feature in 4.5 is the addition of the new Explain feature in Graph*i*QL, visible by using the `--allow-explain` flag. In Graph*i*QL you'll see a new section in your return statement, plus new panel on the right which will show executed query in a readable format.
+
+<div class="tc">
+<img alt="Demo of the new GraphiQL Explain feature in PostGraphile" src="/images/graphiqlexplainbutton.png" />
+</div>
+
+### Smart Tags
+
+No longer do you need to put `COMMENT` commands in the database ("smart comments") to customise your schema, you can now use a tags file to gather these "smart tags" into one, version controlled, easy-to-diff place. Smart comments and smart tags can be used in unison, and both will be supported going forward so you can pick whichever works best for your team.
+
+### Thank you for your contributions
```

Comment:
Please acknowledge the code contributors too; there was quite a lot in this release!

Reply:
How's this? 

### Big contributions from the Graphile community

Thanks to everyone in the Graphile community who has contributed to this release; it's been a good release for community involvement as you can see in the [release notes](https://github.com/graphile/postgraphile/releases/tag/v4.5.0). Contributions from the community include improvements to QueryBuilder, better JWT support and bug fixes for types.